explorer/.flowconfig

```diff
@@ -0,0 +1,12 @@
+[ignore]
+
+[include]
+
+[libs]
+flow-typed
+
+[lints]
```

Comment:
Okay for now. I really want to get ESLint set up, and preferably also as a blocking CI. But that can come later.